spec/index.bs

```diff
@@ -416,12 +416,12 @@ Issue(gpuweb/gpuweb#514): Document read-only states for depth views.
 
 For each [=subresource=] of a [=physical resource=], the usage is tracked on the [=Queue timeline=].
 
-Issue: This section will need to be redacted to support multiple queues.
+Issue: This section will need to be revised to support multiple queues.
 
-The [=Queue timeline=] could be split into a sequence of [=usage scopes=]:
-within each scope the usage of the [=subresource=] stays unchanged,
-and the implementation transitions the [=subresource=] to the new usage
-at the [=usage scope=] boundaries.
+The [=Queue timeline=] is split into a sequence of [=usage scopes=].
```

Comment:
I don't think saying "is split" reflects the semantics here either. It's basically a view. We can view the queue timeline this way, it doesn't mean that the timeline *is* a sequence of scopes.

Reply:
Revised. How's this?